### ISSUE TITLE
The plot of keypoint labels keep scaling down in pose detection training when epoch number increases

### ISSUE THREAD

**Issue by raaachli** [ultralytics/ultralytics#2180]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

When training a pose detection model, as I increase the epoch number, the plotting of the validation label keypoints keeps scaling down. For example, when the model is trained for one epoch, the val_batch0_labels.jpg is:
<img width="651" alt="Screenshot 2023-04-21 at 4 13 28 PM" src="https://user-images.githubusercontent.com/37092132/233726430-361c7196-6013-4a13-b02c-8777e3a8f8a7.png">

When trained for 10 epochs, the the val_batch0_labels.jpg looks like this:
<img width="650" alt="Screenshot 2023-04-21 at 4 14 26 PM" src="https://user-images.githubusercontent.com/37092132/233726581-7342aa50-f72a-4206-bb78-29657b58cb23.png">

Seems like the positions of the keypoints are preprocessed multiple times as the epoch number increases.

### Environment

-Ultralytics YOLOv8.0.71 🚀 Python-3.10.9 torch-1.13.1+cu117 CUDA:0 (NVIDIA GeForce RTX 3090 Ti, 24239MiB)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
Hello @raaachli, thank you for bringing this to our attention. It seems that the scaling of the keypoints is decreasing as the number of epochs increases. This suggests that the model may be learning an incorrect scaling function. You could try adjusting your model's hyperparameters, such as learning rate or optimizer, to see if it corrects the issue. Another suggestion would be to revisit and double-check your data preprocessing steps, scaling, and augmentation, to ensure they are correctly applied consistently. If these suggestions do not resolve the issue, please provide more details about your model architecture and training process to aid in debugging.

**Comment by Laughing-q** [comment]
@raaachli hi, this is a known bug in our previous version, and we've already fixed it in our latest version. please upgrade the package version and try again. Thanks!
